test/Lokad.ILPack.Tests/Lokad.ILPack.Tests.csproj

```diff
@@ -11,8 +11,19 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="3.0.0" />
-    <PackageReference Include="xunit" Version="2.4.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="CS-Script" Version="3.29.0" />
```

Comment:
This seems to be an alternative to `Microsoft.CodeAnalysis.CSharp.Scripting`, right ? I suggest to move it to a conditional `ItemGroup`.

Reply:
Yes, this is a replacement. I've moved it.